roles/openstack-setup/tasks/cinder.yml

```diff
@@ -10,7 +10,7 @@
                       extra_specs="volume_backend_name={{ item.key }}"
                       insecure={{ insecure|default(omit) }}
   when:
-    - ceph.enabled | default('False') | bool
+    - ceph.enabled
```

Comment:
As we have ceph.enabled = false in defaults-2.0.yml, I don't want redundancy here